src/main/java/uk/gov/companieshouse/reconciliation/function/compare_collection/CompareCollectionRoute.java

```diff
@@ -29,6 +31,16 @@ public void configure() throws Exception {
                 .simple("${header.Target}")
                 .bean(CompareCollectionTransformer.class)
                 .marshal().csv()
+                .toD("${header.Upload}")
+                .toD("${header.Presign}")
+                .setHeader("CompareCollectionLink", body())
```

Comment:
Can be simplified by using a single header to store all values and descriptions across all pipelines.